src/adapters/storage.js

```diff
@@ -21,6 +21,18 @@ export default storage => {
     },
     beforeSet(data) {
       return typeof data === 'string' ? data : JSON.stringify(data)
-    }
+    },
+    onGetError(error) {
```

Comment:
Can you replace `error` with `err` please? This would remove any confusion with the `Error` constructor.